images/node/Dockerfile

```diff
@@ -10,14 +10,14 @@
 #
 FROM openshift/origin
 
+COPY centos-paas-sig-openshift-origin-14.repo /etc/yum.repos.d/
 COPY usr/bin/* /usr/bin/
 COPY opt/cni/bin/* /opt/cni/bin/
 COPY etc/cni/net.d/* /etc/cni/net.d/
 COPY conf/openshift-sdn-ovs.conf /usr/lib/systemd/system/origin-node.service.d/
 COPY scripts/* /usr/local/bin/
 
-RUN curl -L -o /etc/yum.repos.d/origin-next-epel-7.repo https://copr.fedoraproject.org/coprs/maxamillion/origin-next/repo/epel-7/maxamillion-origin-next-epel-7.repo && \
-    INSTALL_PKGS="libmnl libnetfilter_conntrack openvswitch \
+RUN INSTALL_PKGS="libmnl libnetfilter_conntrack openvswitch \
```

Comment:
`ose/enterprise-3.4` (and all newer versions of this code) has `conntrack-tools` here -- unclear if we need it